config/filter.d/nginx-wplogin.conf

```diff
@@ -0,0 +1,15 @@
+# Fail2Ban filter for Nginx Web Server wp-login brute force attacks
+#
+#
+[INCLUDES]
+
+# Read common prefixes. If any customizations available -- read them from
+# common.local
+before = common.conf
+
+[Definition] 
+
+failregex = ^<HOST> - .*] "POST /wp-login.php HTTP/1.1" 500
```

Comment:
Never do that. It's very vulnerable (so it would be not anchored at all, neither at end but nor at begin of expression)
Just take a look by another nginx filters how it will be made, e. g. https://github.com/fail2ban/fail2ban/blob/0.10/config/filter.d/nginx-botsearch.conf#L11

Additionally it may be very slow and resources hungry on long lines, that normally not match (because regex will try to follow greedy match `.*` as long as possible).
BTW. It will work only for method "POST" and "HTTP/1.1", so attacker can easy bypass it by using another method or http-protocol.

So your expression would be like:
```diff
- failregex = ^<HOST> - .*] "POST /wp-login.php HTTP/1.1" 500
+ failregex = ^<HOST> \- \S+ \[[^\]]*\] \"(\w+) /wp-login.php \S+\" 500
```